index.js

```diff
@@ -96,7 +101,7 @@ app.on("ready", () => {
 
     const menu = new Menu();
 
-    globalShortcut.register("CommandOrControl+Shift+g", () => {
+    globalShortcut.register("CommandOrControl+Alt+g", () => {
```

Comment:
I would be cautious about making this change, as it isn't backwards-compatible and people may already be accustomed to the current shortcut. Additionally, I believe the new one is already used by Google Drive.